internal/connector/internal/handlers/connectors.go

```diff
@@ -389,6 +390,27 @@ func validateConnectorPatch(bytes []byte, ct *dbapi.ConnectorType) *errors.Servi
 	return nil
 }
 
+func validateConnectorImmutableProperties(resource public.Connector, originalResource public.Connector) handlers.Validate {
```

Comment:
have you checked that these are the only immutable properties? I seems to remember there was some logic in the patch method implementation to allow only some specific parts to be patchable like https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/connector/internal/handlers/connectors.go#L228

Reply:
Thanks @valdar for making me double check.

I effectively had the validation inverted. It should be better now.